src/Prima/Pyxis/Accordion.elm

```diff
@@ -295,24 +266,24 @@ render (State isOpen) ((Config { type_, tagger, slug, title, content }) as confi
     in
     div
         ([ id slug
-         , [ ( "a-accordion", True )
-           , ( "a-accordion--base", isBase type_ )
-           , ( "a-accordion--light", isLight type_ )
-           , ( "a-accordion--dark", isDark type_ )
+         , [ ( "accordion", True )
+           , ( "accordion--base", isBase type_ )
+           , ( "accordion--light", isLight type_ )
+           , ( "accordion--dark", isDark type_ )
            , ( "is-open", isOpen )
            ]
             |> buildWrapperClass config
          ]
             |> (++) options.attributes
         )
         [ div
-            [ class "a-accordion__toggle fs-xsmall fw-heavy a-link--alt"
+            [ class "accordion__toggle"
             , onClick (tagger slug isOpen)
             ]
             [ title
                 |> Maybe.withDefault (text "")
-            , i
-                [ buildIconClass config ]
+            , div
```

Comment:
come mai div che wrappa icona? non sarebbe meglio span?

Reply:
Stiamo cambiando tutte le icone in <i>, entro pranzo pusho ;)